yarn.lock

```diff
@@ -1159,6 +1159,45 @@
   dependencies:
     "@jridgewell/trace-mapping" "0.3.9"
 
+"@der-freitag/mockup@5.1.4":
+  version "5.1.4"
+  resolved "https://gitlab.com/api/v4/projects/52336424/packages/npm/@der-freitag/mockup/-/@der-freitag/mockup-5.1.4.tgz#922515485840f3e2995eec0c8e641525ee33a38b"
+  integrity sha1-kiUVSFhA8+KZXuwMjmQVJe4zo4s=
+  dependencies:
+    "@11ty/eleventy-upgrade-help" "2"
+    "@patternslib/pat-code-editor" "4.0.1"
+    "@patternslib/patternslib" "^9.9.5"
+    backbone "1.4.1"
+    backbone.paginator "2.0.8"
+    bootstrap "5.3.1"
+    bootstrap-icons "1.10.5"
+    cs-jqtree-contextmenu "0.1.0"
+    datatables.net "1.12.1"
+    datatables.net-bs5 "1.12.1"
+    datatables.net-buttons "2.2.3"
+    datatables.net-buttons-bs5 "2.2.3"
+    datatables.net-colreorder "1.5.6"
+    datatables.net-colreorder-bs5 "1.5.6"
+    datatables.net-fixedcolumns "4.1.0"
+    datatables.net-fixedcolumns-bs5 "4.1.0"
+    datatables.net-fixedheader "3.2.4"
+    datatables.net-fixedheader-bs5 "3.2.4"
+    datatables.net-rowreorder "1.2.8"
+    datatables.net-rowreorder-bs5 "1.2.8"
+    datatables.net-select "1.4.0"
+    datatables.net-select-bs5 "1.4.0"
+    dropzone "4.3.0"
+    jqtree "1.7.0"
+    jquery "^3.7.0"
+    jquery-form "4.3.0"
+    jquery.browser "0.1.0"
+    js-cookie "^3.0.5"
+    select2 "git+https://github.com/ivaynberg/select2.git#3.5.4"
+    sortablejs "^1.15.0"
+    tinymce "^5.10.7"
+    tinymce-i18n "^22.12.27"
```

Comment:
We should see if this ends up in the final bundle, otherwise, we should also remove tinymce references from `mockup`'s `package.json` 🤔

Reply:
it doesnt seem to be in the final bundle! :+1: